src/collective/glossary/browser/static/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.css
```

Comment:
this file is not necessary, just ignore the whole static folder in the root .gitignore file